### PR TITLE
s3 file upload

### PR DESCRIPTION
**Pull request by Jakypbek**
added 3 classes and add an amazon dependesie, for use a upload pickture call a Image service there some named methods

### CODE REVIEW

File: src/main/java/kg/airbnb/airbnb/apis/ImageApi.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import org.springframework.core.io.ByteArrayResource;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/file")
+public class ImageApi {
```
Comment (peaksoft-school) [inline src/main/java/kg/airbnb/airbnb/apis/ImageApi.java:15]: Where Cross-Origin?

File: src/main/java/kg/airbnb/airbnb/apis/ImageApi.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import lombok.RequiredArgsConstructor;
+import org.springframework.core.io.ByteArrayResource;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/file")
```
Comment (peaksoft-school) [inline src/main/java/kg/airbnb/airbnb/apis/ImageApi.java:14]: add "/api/file"

File: src/main/java/kg/airbnb/airbnb/apis/ImageApi.java (hunk 1 above)
Comment (peaksoft-school) [inline src/main/java/kg/airbnb/airbnb/apis/ImageApi.java:15]: Rename class FileApi

**Review by peaksoft-school — COMMENTED** [review]
Commented 03/08/22 at 9.40 pm

**Comment by beksultancs** [comment]
@Jakypbek At First Learn How To Upload File to S3* Learn at documentation*
this is trash code!

**Review by beksultancs — COMMENTED** [review]
Eki task bolup 
bolunsun*

**Pull request closed without merging**